lib/src/dom.dart

```diff
@@ -271,41 +265,31 @@ class Node {
 
   Node(this._map);
 
-  Map<String, String> _attributes;
-
-  Map<String, String> get attributes {
-    if (_attributes == null && _map.containsKey('attributes')) {
-      _attributes = _attributeListToMap((_map['attributes'] as List).cast());
-    }
-    return _attributes;
-  }
-
-  int get childNodeCount => _map['childNodeCount'] as int;
+  late final Map<String, String>? attributes = _map.containsKey('attributes')
+      ? _attributeListToMap((_map['attributes'] as List).cast())
+      : null;
 
-  List<Node> _children;
+  int? get childNodeCount => _map['childNodeCount'] as int?;
```

Comment:
Can this actually be null? should we set it to 0 as a default?

Reply:
`children` can be null and then I believe this is `null` also - but in any case it is listed as optional in the protocol https://chromedevtools.github.io/devtools-protocol/tot/DOM/#type-Node